haiku-apps/bamkeys/bamkeys-1.0.0.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Spatially map your display to navigate windows \
+with the keyboard"
+DESCRIPTION="BamKeys let's you break your display up into \
+a grid of zones. Each zone is assigned a keyboard combination. \
+
+Whenever you press that keyboard combination, BamKeys will focus \
+the front-most window that intersects with that zone, regardless \
+of the program it belongs to! Pressing the same zone's keyboard \
+combination repeatedly will focus widows further back. If you have \
+three windows that intersect a zone, pressing that zones hotkey three \
+times will cycle through all those windows. \
+
+Configuring - \
+Open up the BamKeys Application (it's a preference tool) At this point, \
+you'll see a little display with a 3x3 grid, a check box, two pop up \
+menus, and a bunch of key cap looking things. \
+
+The default key mappings for this 3x3 grid are: the left "command" (or alt) \
+key + numeric keypad 1 through 9 (with NumLock ON). If you'd like to \
+change these settings you can - \
+
+• Using the mouse: \
+1. Click the zone on the little display you want to change. It will darken. \
+2. Press (and release) the key combination you want to use. \
+3. Click the zone again to de-select it. \
+
+• Using the keyboard: \
+1. Tab until the screen is selected. \
+2. Press the space-bar. A zone will darken in the grid. \
+3. You can use the arrow keys to change the selected zone. \
+4. Pressing (and releasing) a keyboard shortcut will set the hotkey for that zone. \
+5. Pressing space bar will unselect all zones."
+HOMEPAGE="https://github.com/bvarner/bamkeys"
+COPYRIGHT="2016 Bryan Varner"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev=""
```

Comment:
As the author of bamkeys appears to be absent, I'd recommend filling the latest commit here then include your patchset in `PATCHES`